istioctl/pkg/multicluster/cluster.go

```diff
@@ -61,153 +36,6 @@ func clusterUID(client kubernetes.Interface) (types.UID, error) {
 	return kubeSystem.UID, nil
 }
 
```

Comment:
keep those in the cluster.go file that are still used/referenced by other places in the codebase